fiboa_cli/convert_utils.py

```diff
@@ -432,28 +441,29 @@ def convert(self, output_file, cache=None, input_files=None, source_coop_url=Non
         if directory:
             os.makedirs(directory, exist_ok=True)
 
-        urls = self.get_urls()
         if input_files is not None and isinstance(input_files, dict) and len(input_files) > 0:
             log("Using user provided input file(s) instead of the pre-defined file(s)", "warning")
             urls = input_files
-        elif urls is None:
+        elif (urls := self.get_urls(cache=cache)) is None:
```

Comment:
It seems the only thing that is ever passed is cache, but the methods accepts kwargs. So to understand what's actually passed in here, you need to look through the code. Wouldn't it make more sense to use cache instead of kwards in those get_urls implementations?

Reply:
Even cache is not used. I made this overridable "hook"-function overly generic so we could later add new parameters without changing the implementations. But YAGNI and let's get rid of it all...

Reply:
Fixed "assignment in expression"